hw4.Rmd

```diff
@@ -30,43 +37,103 @@ Before doing anything else, just play with the data to get a sense of it.
 (@) The main outcome of interest is *decision* which records whether the judge granted an appeal or not.  In what proprotion of cases did the judge grant an appeal.
 
 ```{r}
-# [your code here]
+# languages less likely to have an appeal granted?
+dat %>%
+  select(language, decision) %>%
+  group_by(language) %>%
+  summarise(yes = mean(decision=="yes")) %>%
+  ggplot(aes(x=language, y=yes)) +
+    geom_bar(stat="identity") +
+    ylab("percent appealed granted")
+
+# cases vs. percent appeal granted per nation:
+dat %>%
+  select(nation, decision) %>%
+  ggplot(aes(x=nation, fill=decision)) +
+    geom_bar() +
+    theme(axis.text.x=element_text(angle=90, hjust=1))
+
 ```
 
 (@) There are 12 different judges in the data. A key question is whether different judges have different rates of granting an appeal.  Make a plot showing the number of cases per judge.
 
 ```{r}
-# [your code here]
+# are the judges fair 50/50 in granting appeal?
+dat %>%
+  select(judge, decision) %>%
+  group_by(judge) %>%
+  summarise(cases = sum(decision)) %>%
```

Comment:
You don't need to do a lot of these steps. using geom_bar with the default stat of "bin" instead of "identity" eliminates the need for you to do additional calculations. See solutions.